test/runtests.jl

```diff
@@ -7,22 +7,22 @@ for distributed in (true, false)
     ts = @testset RAITestSet "outer" distributed=distributed begin
         @testset "middle" begin
             @testset TestRelTestSet "inner" begin
-                @test (sleep(1); true)
+                @test (sleep(2); true)
```

Comment:
Run as a github action, this test usually takes slightly more than 2s. I bumped up the time and the accepted range to both let it finish and still catch miscounting.